resources/whitelist.tz

```diff
@@ -0,0 +1,289 @@
+# SPDX-FileCopyrightText: 2020 TBD
```

Comment:
I think it's wrong, the license should be taken from https://github.com/tqtezos/lorentz-contract-whitelist

Reply:
license was updated

Reply:
@gromakovsky ping